plugins/sonarqube/src/components/SonarQubeCard/RatingCard.tsx

```diff
@@ -50,29 +50,33 @@ export const RatingCard = ({
   rightSlot,
   title,
   titleIcon,
+  link,
 }: {
   leftSlot: ReactNode;
   rightSlot: ReactNode;
   title: string;
   titleIcon?: ReactNode;
+  link: string;
 }) => {
   const classes = useStyles();
 
   return (
-    <Grid item className={classes.root}>
-      <Grid item className={classes.upper}>
-        <Grid item className={classes.left}>
-          {leftSlot}
+    <a href={link}>
```

Comment:
Curious, what is the tradeoff between using `<a>` vs MUI's `<Link>` component?

Reply:
When using MUI Link you need to customize the color and the underline property:
`      <Link href={link} color="inherit" underline="none">
`
I could also replace it, if you prefer...

Reply:
Styling and a few extra utility props, but we should use it

Reply:
@jeskosda please update 👍

Reply:
Done 👍